Assets/Scripts/HandController.cs

```diff
@@ -26,6 +32,7 @@ public class HandController : MonoBehaviour {
         cardGameObject.name = cardName;
         // Give the card object's script a reference to this script.
         cardGameObject.GetComponent<CardController> ().handController = this;
+        cardGameObject.GetComponent<CardController> ().spell = ReadJsonToSpell (spellPaths[index]);
```

Comment:
Obviously this is just a test line and will change, but I thought this was funny:

FYI this throws an OutOfBounds error if you put >= 5 cards on the board.

Why not >= 4? Because `spellPaths` also contains `Assets/TextMesh Pro/Sprites/EmojiOne.json` xD